defi/src/protocols/data3.ts

```diff
@@ -62195,7 +62195,6 @@ const data3: Protocol[] = [
     twitter: "sacra_fi",
     forkedFrom: [],
     oracles: [],
-    parentProtocol: "parent#tetu",
```

Comment:
Why remove this?

Reply:
Because its different projects

Reply:
@Define101 I can return if its important

Reply:
@alexandersazonof we also group projects together when they are part of the same team. I notice that they are all being worked on from the tetu github repo

Reply:
return parent protocol

Reply:
@realdealshaman ping